File: tuts/012-transitgateway-gettingstarted/go/scenarios/getting_started.go
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package main

import (
	"context"
	"fmt"
	"github.com/aws/aws-sdk-go-v2/config"
	aws_ec2 "github.com/aws/aws-sdk-go-v2/service/ec2"
)

func main() {
	cfg, _ := config.LoadDefaultConfig(context.TODO())
	client := aws_ec2.NewFromConfig(cfg)
	_ = client
	fmt.Println("Running Ec2 getting started scenario...")
	// TODO: setup, interact, teardown
}

File: tuts/012-transitgateway-gettingstarted/javascript/scenarios/getting-started.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { EC2Client } from "@aws-sdk/client-ec2";
import { randomUUID } from "crypto";

const main = async () => {
  const client = new EC2Client({});
  const suffix = randomUUID().slice(0, 8);
  console.log("Running ec2 getting started scenario...");
  // TODO: implement setup, interact, teardown
  console.log("Scenario complete.");
};

main();
